project/ws/viewer/src/lib/plugins/html/html.component.ts

```diff
@@ -202,7 +209,6 @@ export class HtmlComponent implements OnInit, OnChanges, OnDestroy {
   }
 
   getThreshold() {
-    this.tocConfig = this.widgetContentSvc.tocConfigData
```

Comment:
it should be removed?

Reply:
yes, earlier it was coming from other component and it was initialized in widgetContentService, but now in UI rewamp that component is not used so data was undefined. Now reading tocConfig from subscribe event